gozstd.go

```diff
@@ -308,6 +331,9 @@ func decompressInternal(dctx, dctxDict *dctxWrapper, dst, src []byte, dd *DDict)
 	// Prevent from GC'ing of dst and src during CGO calls above.
 	runtime.KeepAlive(dst)
 	runtime.KeepAlive(src)
+	runtime.KeepAlive(dctx)
```

Comment:
It is better to create a separate pull request for these changes, since they aren't related to the decompression with the limited size of the decompressed data.

Reply:
Removed, it was accidental change.